readme.md

```diff
@@ -1,9 +1,8 @@
-# Code style guide for [CKEditor](https://github.com/ckeditor/ckeditor-dev)
+# 1. Introduction
```

Comment:
So, in that case, the lowest heading would have 5 dots which is plenty (`1.1.1.1.1.`). I was trying to avoid such long nesting :thinking: Maybe we could go with just additional header at the top (so restoring previous `Code style guide for CKEditor` without introducing numbering for that? Not sure if it's correct (probably not) :thinking:

Reply:
Ok seems like I incorrectly understood your comment. It's about changing headers level only, so :+1: